src/pages/trails/ballots/list/ballots-list.vue

```diff
@@ -1,242 +1,209 @@
 <script>
-import { mapActions, mapGetters, mapMutations } from "vuex";
-import BallotForm from "../components/ballot-form";
-import BallotListItem from "../components/ballot-list-item";
-import BallotView from "../view/ballot-view";
-import WelcomeCard from "../../../../components/welcome-card";
-import ActionBar from "../../../../components/action-bar";
+import { mapActions, mapGetters, mapMutations } from 'vuex'
+import BallotForm from '../components/ballot-form'
+import BallotListItem from '../components/ballot-list-item'
+import BallotView from '../view/ballot-view'
+import WelcomeCard from '../../../../components/welcome-card'
+import ActionBar from '../../../../components/action-bar'
```

Comment:
there is an alias, `~` for `src` e.g.  ~/components/action-bar` use that instead of relative path.

Reply:
App •  ERROR  •  UI  in ./src/pages/trails/ballots/list/ballots-list.vue?vue&type=script&lang=js

Module not found: Can't resolve imported dependency "~/view/ballot-view"
Did you forget to install it? You can run: yarn add ~/view/ballot-view